src/renderers/dom/server/ReactServerRenderingTransaction.js

```diff
@@ -7,12 +7,14 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactServerRenderingTransaction
+ * @flow
```

Comment:
It passes but you did not type all the methods. I think you can check this with `flow coverage`. 

Just passing is not enough. Flow passing doesn't guarantee that it typed everything, it just means it hasn't found violations so far.

The constraint we are operating under is that we either type the file completely, or we don't type it at all.